src/core/Akka.Cluster/Member.cs

```diff
@@ -264,11 +263,62 @@ public int Compare(Member x, Member y)
         }
 
         /// <summary>
+        /// Combines and sorts two lists of <see cref="Member"/> into a single list ordered by Member's valid transitions
+        /// </summary>
+        /// <param name="a">The first collection of members.</param>
+        /// <param name="b">The second collection of members.</param>
+        /// <returns>An immutable hash set containing the members with the next logical transition.</returns>
+        public static ImmutableSortedSet<Member> PickNextTransition(IEnumerable<Member> a, IEnumerable<Member> b)
```

Comment:
Implemented a pair-wise merging mechanism for two collections containing some of the same members, but with different statuses. Used in the MBT from earlier and the other `MemberOrderingSpec` I added.